.gitignore

```diff
@@ -22,4 +22,7 @@ _testmain.go
 *.exe
 *.test
 *~
-*.DS_Store
\ No newline at end of file
+*.DS_Store
+
+*_got
```

Comment:
Why do we need this ignore lines now?

Reply:
when spaceinviders tests fail it generates the `<name>_got` and `<name>_expected` svg files, I got plenty of them while was fixing tests, so it made sense to add this files to the `.gitignore` to accidentally not commit them.

Reply:
`*.DS_Store` - I think VS Code automatically added it.